app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicListFragment.kt

```diff
@@ -13,7 +13,7 @@ class OngoingTopicListFragment : InjectableFragment() {
 
   companion object {
     // TODO(#1655): Re-restrict access to fields in tests post-Gradle.
-    const val ONGOING_TOPIC_LIST_FRAGMENT_TAG = "TAG_ONGOING_TOPIC_LIST_FRAGMENT"
+    const val ONGOING_TOPIC_LIST_FRAGMENT_TAG = "OngoingTopicListFragment.tag"
```

Comment:
Please take a look at this comment:
#1717 (comment)

Reply:
done